pygmt/src/grdhisteq.py

```diff
@@ -163,7 +163,7 @@ def equalize_grid(
         ----------
         grid : str or xarray.DataArray
             The file name of the input grid or the grid loaded as a DataArray.
-        outgrid : str or bool or None
+        outgrid : str or None
```

Comment:
Need to change L144: should be `outgrid=None`

Reply:
Thanks, fixed.